src/wallet/rpcwallet.cpp

```diff
@@ -3958,7 +3958,7 @@ UniValue listissuances(const JSONRPCRequest& request)
             "listissuances ( asset ) \n"
             "\nList all issuances known to the wallet for the given asset, or for all issued assets if none provided.\n"
             "\nArguments:\n"
-            "1. \"asset\"                 (string, optional) The asset whose issaunces you wish to list.\n"
+            "1. \"asset\"                 (string, optional) The asset whose issaunces you wish to list. Accepts both the asset hex or the locally assigned asset label.\n"
```

Comment:
sorry grammar nit: "both ... or". Should be "either ... or" or "both ... and". "Either or" seems more logical to me since it's one or the other.